user_profile/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
```

Comment:
lets create a directory "forms" and create separate files for each form.

Reply:
tried but it breaks the app.
in user_profile.forms.UserProfileForm, user_profile.forms is not a valid app